10/debian-10/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -495,6 +495,29 @@ pg_ctl_options='-o "--config-file=\"${POSTGRESQL_CONF_FILE}\" --external_pid_fil
 pg_basebackup_options='$waldir_option'
 EOF
 
+	if [[ "$REPMGR_FENCE_OLD_PRIMARY" == "true" ]]; then
```

Comment:
This could be changed to:
```suggestion
        if is_boolean_yes "$REPMGR_FENCE_OLD_PRIMARY"; then
```
Take a look to my comment in `postgresql-env.sh`.